### ISSUE TITLE
Iterative grid search

### ISSUE THREAD

**Issue by bafflerbach** [uw-cmg/MAST-ML#161]
Be able to choose a number of iterations that pick most promising regions to grid more densely over smaller area and run again.

**Comment by qpwo** [comment]
Yes this is an excellent idea! It might be really hard to implement. We can also have it automatically stop making grids and shrinking smaller areas once the model stops improving.
